driver-moab/main/prep_atm_mod.F90

```diff
@@ -221,6 +224,16 @@ subroutine prep_atm_init(infodata, ocn_c2_atm, ice_c2_atm, lnd_c2_atm, iac_c2_at
       if (trim(atm_gnam) /= trim(lnd_gnam)) samegrid_al = .false.
       if (trim(atm_gnam) /= trim(ocn_gnam)) samegrid_ao = .false.
 
+#ifdef HAVE_MOAB
+      wgtIdo2a = 'conservative_o2a'//C_NULL_CHAR
+      wgtIdi2a = 'conservative_i2a'//C_NULL_CHAR
+      wgtIdl2a = 'conservative_l2a'//C_NULL_CHAR
```

Comment:
Why not just "conservative"?  Does conservative_o2a mean a different algorithm from conservative_l2a ?

Reply:
Answer:   wgtIdo2a becomes a hash so needs to be unique from wgtIdi2a.